profiles/w3c.js

```diff
@@ -56,6 +56,7 @@ const modules = [
   import("../src/core/data-tests.js"),
   import("../src/core/list-sorter.js"),
   import("../src/core/highlight-vars.js"),
+  import("../src/core/dfn-panel.js"),
```

Comment:
Will add to other profile also. Not sure if this should be module of it's own or a part of `core/dfn`